tests/cookbooks/harness.py

```
@@ -0,0 +1,101 @@
+import os
+from pathlib import Path
+
+import httpx
+from fishaudio import AsyncFishAudio as _AsyncFishAudio
+from fishaudio import FishAudio as _FishAudio
+
+BASE_URL = "https://api.fish.audio"
+# Validated public voice ("Energetic Male"), used to fill <voice-id> placeholders.
+PUBLIC_VOICE = "802e3bc2b27e49c2995d23ef70e6ac89"
+_WORKSPACE_ENV = "/Users/shawnlai/project/fish-audio/.env"
+_LOCAL_KEYFILE = "/tmp/claude/fishdoctest/fishkey"
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/claude/fishdoctest/fishkey"

(S108)